lib/generators/react_on_rails/base_generator.rb

```diff
@@ -62,7 +62,7 @@ def update_git_ignore
           /app/assets/javascripts/generated/*
         DATA
 
-        dest_file_exists?(".gitignore") ? append_to_file(".gitignore", data) : puts_setup_file_error(".gitignore", data)
+        dest_file_exists?(".gitignore") ? append_to_file(".gitignore", data) : options.errors.push(return_setup_file_error(".gitignore", data))
```

Comment:
How about if we create a singleton of our own class to that supports an api to add errors and later can be called to get the errors.

Putting errors on the options is really hacky.